CHANGELOG-2.3.md

```diff
@@ -0,0 +1,17 @@
+# Changelog since v2.2.0
+
+## New Features
+
+- Add '--csi.testvolumeexpandsize` flag ([#219](https://github.com/kubernetes-csi/csi-test/pull/219), [@wnxn](https://github.com/wnxn))
+
+
+## Bug Fixes
```

Comment:
I think https://github.com/kubernetes-csi/csi-test/pull/217 is worthy of a release note. I updated the PR with one. Could you add it here?

Reply:
Done. While looking at the changelog again I noticed that capitalization and past vs. present tense was inconsistent; it's a bit better now.

Reply:
still don't see it?

Reply:
Darn, missing `-a` in `git commit --amend`. Done now.